### ISSUE TITLE
Error creating machine: Error in driver during machine creation [ boot2docker under xhyve ]

### ISSUE THREAD

**Issue by alexbrebu** [codekitchen/dinghy#281]
occurs on `macOS Mojave 10.14.2`

`Homebrew` used to install: dinghy, docker, docker-compose, docker-machine, docker-machine-nfs, docker-machine-driver-xhyve
***
codekitchen/dinghy/dinghy 4.6.5
docker 18.09.0
docker-compose 1.23.2
docker-machine 0.16.0
docker-machine-driver-xhyve 0.3.3_1
***
![screenshot 2018-12-10 at 17 28 43](https://user-images.githubusercontent.com/41296804/49742372-152cec80-fca1-11e8-95c5-a13937327281.png)

**Comment by alexbrebu** [comment]
Meanwhile, recommend to use an older version for boot2docker by specifying `--boot2docker-url` 
`dinghy create --provider=xhyve --cpus=2 --memory=8192 --disk=80000 --boot2docker-url=https://github.com/boot2docker/boot2docker/releases/download/v17.12.1-ce/boot2docker.iso`

**Comment by alexbrebu** [comment]
:white_check_mark:: dinghy create --provider=xhyve --cpus=${cpus:-3} --memory=${memory:-4096} --disk=${disk:-40000} --boot2docker-url=https://github.com/boot2docker/boot2docker/releases/download/v18.06.1-ce/boot2docker.iso
:rocket:: eval $(dinghy env)

**Comment by alexbrebu** [comment]
@codekitchen

**Comment by beardcoder** [comment]
Yes i can reproduce that. But i thing this is a problem with ne driver and boot2docker

**Comment by codekitchen** [comment]
This does look like a problem with boot2docker or the xhyve driver. Do you get the same error if you do a `docker-machine create -d xhyve` directly?

On your workaround, was there a particular reason that you chose the v18.06.1-ce boot2docker version? I'm curious why that would fix it.

If this does happen when running `docker-machine create` directly also, I'd recommend reporting it to the docker-machine-driver-xhyve project. I'll see if I can repro as well.

**Comment by alexbrebu** [comment]
I did ` docker-machine create -d "xhyve" theMachine` directly.

![Uploading Screenshot 2018-12-19 at 17.36.08.png…]()
***
Workaround came as v18.06.1-ce boot2docker was the last version that I remembered I had tried  before and worked.
***
Have you succeeded to repro?

**Comment by codekitchen** [comment]
I am able to successfully repro, and I found this existing issue in the docker-machine-driver-xhyve project for the issue: https://github.com/machine-drivers/docker-machine-driver-xhyve/issues/211

It sounds like it has already been fixed upstream and the fix will be in the next boot2docker release, so your workaround is the best option for now.

**Comment by beardcoder** [comment]
Yes it will work with 
```
dinghy create --memory=8000 --cpus=4 --disk=100000 --provider=xhyve --boot2docker_url="https://github.com/boot2docker/boot2docker/releases/download/v18.09.1-rc1/boot2docker.iso"
```

**Comment by codekitchen** [comment]
It looks like this is now fixed upstream with boot2docker v18.09.1, and the workaround is no longer necessary.
